src/app/board/board.controller.js

```diff
@@ -109,34 +108,30 @@ export class BoardController {
     //CODE TO ENABLE DRAG AND DROP OF STICKYs
     $('.delete-btn').css({'visibility' : 'hidden'});
     $('.like-btn').css({'visibility' : 'hidden'});
+    $('.edit-btn').css({'visibility' : 'hidden'});
+    $('.save-btn').css({'display' : 'hidden'});
     $("#chat-messages-plus").sortable();
     $("#chat-messages-minus").sortable();
     $("#chat-messages-plus").disableSelection();
     $("#chat-messages-minus").disableSelection();
 
     //CODE TO SHOW DELETE/LIKE ETC ON HOVER
-    this.hover = function(msg){
-      if(msg.uid === userId) {
-
+    this.hover = function(msgId){
+      if(msgId === userId) {
+        $('.edit-btn').css({'visibility' : 'visible'});
```

Comment:
This is done to only show the edit button for the respective user. He shouldn't be able to edit other people's messages.
It's still buggy, but will finish it in some time.